crates/bevy_reflect/src/type_registry.rs

```diff
@@ -599,6 +599,23 @@ impl<T: Reflect> FromType<T> for ReflectFromPtr {
     }
 }
 
+/// In order to dynamic get component's dyn object.
```

Comment:
Nit: We should avoid ECS terminology in this crate
```suggestion
/// Type data for transforming a `dyn Reflect` into a `dyn Trait` trait object.
```